ui/ui-components/pages/model/[modelId]/destination/[destinationId]/edit.tsx

```diff
@@ -1,3 +1,4 @@
+import { DestinationDeliveryMode } from "@grouparoo/core/src/models/Destination";
```

Comment:
We can't import models from core - this will cause compication problems.  The best we can do is import API responses via `import { Actions } from "../../../../../utils/apiData"`